eng/service.proj

```diff
@@ -62,12 +62,6 @@
   <Import Project="..\sdk\$(ServiceDirectory)\*.projects" Condition="'$(ProjectListOverrideFile)' == '' "/>
```

Comment:
I think you will need to remove this condition as well to make it work.

Reply:
We will immediately begin including projects that we don't intend to by including them all. You can see what I'm talking about right?

Reply:
Spoke with @hallipr . The service.projects _should not_ be importing on their own. It should be purely classification. The issue is the few where they do import should be fixed.